docs/shared/reference/environment-variables.md

```diff
@@ -32,3 +32,5 @@ Nx will set the following environment variables so they can be accessible within
 | NX_TASK_TARGET_CONFIGURATION | string  | Set to the configuration name of the task being run. Use this to tell which configuration of the target is being run. |
 | NX_DRY_RUN                   | boolean | Set to `true` during dry runs of generators. Use this to avoid side effects during generators.                        |
 | NX_INTERACTIVE               | boolean | Set to `false` when running generators with `--interactive=false`. Use this to prevent prompting during generators    |
+
+When using distributed caching, [Nx Cloud Environment Variables](/nx-cloud/reference/env-vars) are also available.
```

Comment:
LGTM, potentially we could have it as a callout here as well. Like "Looking for Nx Cloud environment variables? ..."